activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -45,130 +45,780 @@ def primary_key(name, type = :primary_key, **options)
         ##
         # :method: bigserial
         # :call-seq: bigserial(*names, **options)
+        #
+        # Adds a +bigserial+ column, which is a PostgreSQL auto-incrementing 64-bit integer.
+        #
+        # This is commonly used for primary keys or large counters that require a greater range
+        # than +serial+ (which is 32-bit).
+        #
+        # Example:
+        #
+        #   t.bigserial :event_id
+        #
+        # You can also explicitly declare it as the primary key:
+        #
+        #   create_table :events, id: false do |t|
+        #     t.bigserial :id, primary_key: true
+        #   end
+        #
 
         ##
         # :method: bit
         # :call-seq: bit(*names, **options)
+        #
+        # Adds a +bit+ column for storing fixed-length bit strings.
+        #
+        # This type is useful for compactly representing flags or settings as binary values.
+        # By default, you can specify the number of bits the column should store.
+        #
+        # Example:
+        #
+        #   t.column :settings, "bit(8)"  # Stores exactly 8 bits
+        #
+        # When assigning values, use bit strings:
+        #
+        #   irb> user = User.new
+        #   user.settings = "01010101"  # This is a bit string
+        #   user.save!
 
         ##
         # :method: bit_varying
         # :call-seq: bit_varying(*names, **options)
+        #
+        # Adds a +bit_varying+ column for storing variable-length bit strings.
+        #
+        # This type is useful for representing binary flags, feature toggles, or compact
+        # settings as sequences of bits.
+        #
+        # Example:
+        #
+        #   t.bit_varying :settings
+        #
+        # You can also specify the column type explicitly:
+        #
+        #   t.column :flags, 'bit varying(16)'  # Up to 16 bits
+        #
+        # When assigning values, use bit strings:
+        #
+        #   irb> user = User.new
+        #   irb> user.flags = "01010101"  # Valid bit string
+        #   irb> user.save!
+        #
+        # If you assign an integer, it will be converted to a bit string:
```

Comment:
Avoiding "you", maybe something like:
```suggestion
        # When assigning an integer, it will be converted to a bit string:
```